src/modules/job-exec/job-exec.c

```diff
@@ -1381,6 +1394,28 @@ static int job_exec_initialize (flux_t *h, int argc, char **argv)
         }
         flux_log (h, LOG_INFO, "using kill-timeout of %.4gs", kill_timeout);
     }
+    if (ksignal) {
+        if ((kill_signal = sigutil_signum (ksignal)) < 0) {
+            flux_log (h, LOG_ERR, "invalid kill-signal: %s", ksignal);
+            errno = EINVAL;
+            return -1;
+        }
+        flux_log (h,
+                  LOG_INFO,
+                  "using kill-signal of %s",
+                  sigutil_signame (kill_signal));
```

Comment:
Possibly for later, but we did add support for dumping the current config for the exec implementations with `flux module stats` e.g.
```
$ flux module stats job-exec
{
 "bulk-exec": {
  "config": {
   "default_cwd": "/tmp",
   "default_job_shell": "/home/garlick/proj/flux-core/src/shell/flux-shell",
   "exec_service": "rexec",
   "exec_service_override": 0
  }
 }
}
```
the global configs could probably be added at some point and then tests wouldn't have to grep the dmesg buffer.

Reply:
That makes sense I wonder why that wasn't originally added. I can go ahead and add that here since already the changes meant only for testing have taken way longer than the original bug fix. :disappointed: